code/game/atoms.dm

```diff
@@ -750,6 +751,18 @@ its easier to just keep the beam vertical.
 	if(fingerprintshidden && istype(fingerprintshidden))
 		A.fingerprintshidden |= fingerprintshidden.Copy()    //admin	A.fingerprintslast = fingerprintslast
 
+//Atomic level procs to be used elsewhere.
+/atom/proc/apply_luminol(var/atom/A)
```

Comment:
Can replace the whole proc with `return has_blood`

Reply:
Resolved.